spec/spec_helper.rb

```diff
@@ -2,4 +2,9 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  config.mock_with :rspec do |mocks|
+    mocks.syntax = :expect
```

Comment:
@ruslankhaertdinov let's add these option in separate PR b/c

* https://github.com/rspec/rspec-mocks/blob/master/features/old_syntax/README.md
* https://github.com/rspec/rspec-expectations/blob/master/features/syntax_configuration.feature